scaleway/resource_vpc_gateway_network.go

```diff
@@ -189,23 +227,42 @@ func resourceScalewayVPCGatewayNetworkRead(ctx context.Context, d *schema.Resour
 		cleanUpDHCPValue = *expandBoolPtr(cleanUpDHCP)
 	}
 
-	_ = d.Set("gateway_id", newZonedID(zone, gatewayNetwork.GatewayID).String())
-	_ = d.Set("private_network_id", newZonedID(zone, gatewayNetwork.PrivateNetworkID).String())
-	_ = d.Set("enable_masquerade", gatewayNetwork.EnableMasquerade)
+	_ = d.Set("gateway_id", newZonedID(zone, gwn.GatewayID).String())
```

Comment:
Why is this change needed? Using a full variable instead of an acronym is better and would make the diff smaller.

Reply:
Ok, It will be change in next commit